src/utils/utils.js

```diff
@@ -357,8 +362,11 @@ export const broadcastChannelOptions = {
   webWorkerSupport: false, // (optional) set this to false if you know that your channel will never be used in a WebWorker (increases performance)
 }
 
-export function validateVerifierId(selectedTypeOfLogin, value) {
+export function validateVerifierId(selectedTypeOfLogin, value, chainId) {
```

Comment:
chainId might be hex or numeric, currently it only handles for numeric chainIds, we should handle both cases.